libs/btpsa-usecase.json

```diff
@@ -226,11 +226,6 @@
                         "description": "name of the service instance",
                         "title": "name of the service instance"
                     },
-                    "labels":{
```

Comment:
The functionality of labels are removed with your commit - there seems to be something wrong with the rebase/merge